src/views/search/search.jsx

```diff
@@ -95,6 +97,9 @@ var Search = injectIntl(React.createClass({
         }
         return allTab;
     },
+    setSearchTerm: function (searchTerm) {
+        this.props.searchTerm = searchTerm;
```

Comment:
`props` shouldn't be assigned anything outside of an initialization method (like `getDefaultProps`. And, in order to get this into the `navigation` component, you'd need to instead dispatch `setSearchTerm` from `redux/navigation.js`, otherwise, it won't be available as a value outside of this component. Something like `this.props.dispatch(navigationDetails.setSearchTerm(searchTerm))` – but make sure that you [properly hook up redux](https://github.com/LLK/scratch-www/blob/develop/src/views/conference/details/details.jsx#L86-L94) if you do that too.